src/libffi/chevahbs

```diff
@@ -12,7 +12,7 @@ chevahbs_getsources() {
     local name=$1
     local ver=$2
     local ext="tar.gz"
-    local link=ftp://sourceware.org/pub/libffi/"$name"-"$ver"."$ext"
+    local link=https://github.com/libffi/libffi/releases/download/v"$ver"/"$name"-"$ver"."$ext"
```

Comment:
this is ok for this PR.
but for the long term, we should make sure we have full contol over pythia dependencies.

This mean, keeping copies of things like libffi on bin.chevah.com and have chevahbs script download them from bin.chevah.com

Reply:
Sure, no problem. https://github.com/chevah/pythia/issues/33